src/main/java/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java

```diff
@@ -450,7 +449,7 @@ public boolean compareKeysValues(String[] keys, String[] keys2, String[] values,
    * 
    * @throws BadRequestException if the properties parameter contains invalid content
    */
-  public void processPropertiesParam() throws BadRequestException {
```

Comment:
Why did you remove the "throws BadRequestException" here, when this method actually does exactly that potentially?